travis_build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euv
+
+DEPLOY_VERSION=""
+DEPLOY_BRANCH=""
+if [ "$TRAVIS_PULL_REQUEST" == "true" ]; then
```

Comment:
TRAVIS_PULL_REQUEST: The pull request number if the current job is a pull request, “false” if it’s not a pull request.
`if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then` ?

Reply:
This was to test the production part without actually merging ;)